jobs/nfsv3driver/templates/install.erb

```diff
@@ -95,8 +95,27 @@ function prepend_rfc3339_datetime() {
 }
 
 function main() {
-  codename=$(lsb_release -c | grep xenial | awk '{print $2}')
-  if [ "$codename" == "xenial" ]; then
+  codename=$(lsb_release -c | cut -f 2 )
```

Comment:
I doubt this works on windows or centos but the old code didn't care about them either